app/models/barcode.rb

```diff
@@ -21,12 +21,12 @@ def self.calculate_sanger_barcode(prefix, number)
 
     human = prefix + number_s + calculate_checksum(prefix, number)
     barcode = prefix_to_number(prefix) + (number * 100)
-    barcode += human[human.size - 1]
+    barcode += human[human.size - 1] # rubocop:todo Lint/UselessAssignment
   end
 
   def self.calculate_barcode(prefix, number)
     barcode = calculate_sanger_barcode(prefix, number)
-    barcode * 10 + calculate_ean13(barcode)
+    (barcode * 10) + calculate_ean13(barcode)
```

Comment:
Personal opinion only, but I often see linters and developers insisting on brackets that make no difference to the equation.  Maybe this is because people often get the grouping wrong, but I always hate to see excessive brackets.  Multiplication/division always comes before addition/subtraction and even if you only do this equation left to right it's still correctly evaluated.  Is there anything can be done to make Rubocop less fussy about unnecessary brackets?  I'm also happy to have this comment ignored.

Reply:
I think the only option is to disable Lint/AmbiguousOperatorPrecedence entirely:
https://docs.rubocop.org/rubocop/cops_lint.html#lintambiguousoperatorprecedence

On a personal level I agree with you (especially in this particular case) but think its worth leaving the cop in place because:

1) it probably is useful in more extreme cases, particularly where precedence ends up including things like logical operators.
2) I've also noticed other developers insist on it (even in the absence), so can only assume Rubocop is correct in its judgement that it causes ambiguity for some.